<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Happy Coding</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget
{
background-image: url(:/patterns/tex_grey);
background-repeat: repeat-xy;
}

QHBoxLayout#headerLayout
{
background-image: url(:/patterns/carbon_dark);
background-repeat: repeat-xy;
}

QPushButton
{
background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;
color: rgb(122, 182, 10);
}

QPushButton:pressed
{

background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(116, 202, 18);
border-style: outset;
color: green;
}

QWidget#scrollAreaWidgetContents
{
background-image: url(:/patterns/woven);
background-repeat: repeat-xy;
}

QGroupBox {


 border: 1px solid rgb(144,255,23);
 border-radius: 2px; 
 margin-top: 9px;
 margin-bottom: 7px;
 padding: 0px;

background-image: url(:/patterns/woven);
background-repeat: repeat-xy;
} 

QGroupBox::title {
top:-9 ex;
left: 10px;
color: green;
subcontrol-origin: border;

padding: 2px;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;

background-image: 	url(:/patterns/carbon_dark);				
background-repeat: repeat-xy;
}

QScrollArea#scrollAreaEntries
{
margin-left: 9px;
margin-right: 9px;
 border: 1px solid rgb(144,255,23);
}

QLabel#lblTitle
{
padding-left: 9px;
}


QWidget#headerWidget
{
	background: url(:/patterns/carbon_dark);
}

QLabel#headerTitleLbl
{
background: qlineargradient(spread:pad, x1:0.178, y1:0, x2:1, y2:0, stop:0 rgba(37, 202, 0, 234), stop:1 rgba(255, 255, 255, 0));
padding-left: 9px;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="centralWidgetLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="headerWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="headerTitleLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>{ Happy Coding }</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnHelp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>49</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	margin-right: 15px;
	border-radius: 17px;
	border: 3px solid green;
	color: rgb(0, 170, 0);
}

QPushButton:pressed
{
	border-color: rgb(0, 170, 0);
	color: rgb(0, 255, 0);
	font-size: 17px;
}

QPushButton:hover
{
	border-color: rgb(0, 170, 0);
	color: rgb(0, 255, 0);
}</string>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gBoxSearchCriteria">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Eingestellter Suchfilter:</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vertLayout">
        <item>
         <widget class="QPushButton" name="bttnCustomize">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Anpassen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttnSearch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Suchen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bttnNewEntry">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Neuer Eintrag</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaEntries">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>979</width>
         <height>300</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Men√º</string>
    </property>
    <addaction name="actionEinstellungen"/>
    <addaction name="separator"/>
    <addaction name="actionHilfe"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Datenbank laden</string>
   </property>
  </action>
  <action name="actionBeenden_2">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe!</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
